config/settings.yml

```diff
@@ -10,6 +10,7 @@ accredited_representative_portal:
       base_uri: https://api.github.com
       path: <%= ENV['accredited_representative_portal__allow_list__github__path'] %>
       repo: department-of-veterans-affairs/va.gov-team-sensitive
+  frontend_base_url: <%= ENV["accredited_representative_portal__base_url"] || 'http://localhost:3001' %>
```

Comment:
@chumpy @ojbucao I think the way we're setting up these ymls doesn't follow the pattern that is used. I'll make a recommendation very shortly.